### DISCUSSION TITLE
Customizing React Joyride to match a figma design

### DISCUSSION THREAD

**Discussion by deeseeker** [gilbarbara/react-joyride#1119] [Q&A] [answered]
<img width="736" alt="Screenshot 2024-12-23 at 09 07 58" src="https://github.com/user-attachments/assets/dd4bfdd8-fa05-4ba8-940d-879b7e646e13" />

Hi everyone,

I have been playing with react joyride to understand how to customize it to match a given design since from my research it seems to be recommended because it's said to be customizable. After few trials, I was only able to produce what's shown in the screenshot below but still far away from achieving my goals.
<img width="1675" alt="Screenshot 2024-12-23 at 09 10 59" src="https://github.com/user-attachments/assets/79a309e3-3e9b-4d57-a7b3-098d2df19a4e" />

can anyone guide me through how I can customize the whole UI please?

**Comment by gilbarbara** [comment, marked as answer]
Hey @deeseeker 

You probably want to use a custom Tooltip and take complete control of the UI.
Check out the [docs](https://docs.react-joyride.com/custom-components)

**Reply by deeseeker** [reply]
Thanks for your response. I will try it out.
